changelog/fragments/helm-upgrade-rename.yaml

```diff
@@ -0,0 +1,5 @@
+entries:
+  - description: >
+      In Helm-based operators, usages of the verb `Update` were renamed to `Upgrade` to better align with Helm nomenclature.
+    kind: "change"
+    breaking: false
```

Comment:
I do not think that we need a fragment in this case. It does not affect end-users.